windows/ExperimentalFeatures.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Flags can be added here to effect the compilation of Microsoft.ReactNative -->
+  <PropertyGroup Label="Microsoft.ReactNative Build Flags">
+    <UseWinUI3>false</UseWinUI3>
+    <UseHermes>false</UseHermes>
+    <WinUI2xVersion>2.8.0-prerelease.210927001</WinUI2xVersion>
+    <WebView2Version>1.0.1020.30</WebView2Version>
```

Comment:
note that these values won't actually be used except for when building the solution that lives in this folder. When building an app, the app's own experimentalFeatures.props will be used, not this one

Reply:
Yep! Just there for dev env for the module!